src/common/models/Projection.ts

```diff
@@ -173,6 +184,9 @@ export class Projection {
   private readonly smoothedHospitalizations: Array<number | null>;
   private readonly rawICUHospitalizations: Array<number | null>;
   private readonly smoothedICUHospitalizations: Array<number | null>;
+  private readonly weeklyCovidAdmissionsPer100k: Array<number | null>;
+  private readonly bedsWithCovidPatientsRatio: Array<number | null>;
+  private readonly canCommunityLevel: Array<CommunityLevel | undefined>;
```

Comment:
I don't think we need to use the timeseries data (we don't ever graph risk levels in the frontend).  So this can just be `Level` (with us converting `null` => `Level.UNKNOWN`)

Reply:
Would that conversion need to happen here? e.g. with something like `summaryWithTimeseries.communityLevels?.canCommunityLevel ?? Level.UNKNOWN` or does that logic exist elsewhere

Reply:
I think we can / should put it here so we get rid of the nullability as early as possible and downstream code doesn't have to deal with `Level | null` return types.

Reply:
Cool, makes sense! Added changes in 1d88b91f85aad7fd04b32b2d19225184f2f5f1cf. I'm not really sure how enums work in this case, so I'm not sure if the levels automatically map (eg. 0 -> low), or we might need to do that mapping manually?

Reply:
I think it will probably just work.  TypeScript enums are really just `string` or `number` values under the covers and the `Level` enum doesn't explicitly specify the values, so I think they'll be numbers starting at `0`.

Reply:
sweet, makes sense 🙏